.env.example

```diff
@@ -0,0 +1,10 @@
+# Agent Configuration
+AGENT_BASE_URL=http://127.0.0.1:3500
+AGENT_ID=agent_local_qa_agent_id
```

Comment:
This ID can just be what's in the yaml for now right?  Since we're all in the same org, it should work as is.

Reply:
The only folks it won't work for are outside contributors i guess, so their agent IDs would be different.